### PR TITLE
initial-commit

### PR DESCRIPTION
**Pull request by SaadMahi**
Created basic guess the number game using Java script.

### CODE REVIEW

**Review by nehalskdev — APPROVED** [review]
LGTM

File: script.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+'use strict';
```
Comment (murtazshaikh) [inline script.js:1]: Whats the use of "use strict" ?

**Review by murtazshaikh — APPROVED** [review]
Try to use few comments only in JS apart from it looks Amazing! keep it up.

File: script.js (hunk 1 above)
Comment (SaadMahi) [reply script.js:1]: "use strict" is to indicate that the code should be executed in "strict mode" which is useful to execute coding with less errors.

**Pull request merged**